src/locale/localed.c

```diff
@@ -33,6 +33,42 @@
 
 static Hashmap *polkit_registry = NULL;
 
+static int locale_read_data_once(Context *c, sd_bus_message *m) {
+        static sd_bus_message *_m = NULL;
+
```

Comment:
Hmm, this works, true. But maybe it would be better to store this in Context instead, i.e. add a `sd_bus_message *_last_read_for_message`? Context is only used in localed and one test file, so it seems OK to modify the data structure. Then this check could be moved to `locale_read_data`, simplifying the whole thing.